Fixing Grammatical Mistakes (#4006)
magento/devdocs: guides/v2.1/comp-mgr/trouble/cman/out-of-memory.md

  - Upgrade
---

> Sometimes, when you attempt to install or update an extension or the latest Magento 2.3.x version, you can encounter an error due to your system running out of physical memory. We recommend you create a swap file to avoid this error.

Reviewer: ```suggestion
You may encounter an error, caused by your system running out of physical memory, when you attempt to install or update an extension or the latest Magento 2.3.x version. We recommend you create a swap file to avoid this error.
```
